ForumCrawler/Commands/ScoreCommands.cs

```diff
@@ -116,7 +116,6 @@ public async Task GetStats(IUser userObj)
             await ReplyAsync($"[#{hs.Item2}] **{user.GetName()}**'s stats:", embed: new EmbedBuilder().WithDescription(
                 Emote.Parse(WootString) + $" **Score:** {score.Score:F3}{boostStr}\n" +
                 $":gem: **Gems:** {score.Gems}{dailyStr}\n" +
-                $":rocket: **Inertia:** {score.Inertia * 100:F0}%").Build());
```

Comment:
Removing this line prevents the code from compiling. Ensure that the preceding + is removed and whatever else.